source/lib/tabs-service.js

```diff
@@ -26,6 +29,10 @@ export const queryTabs = async url => {
 };
 
 export const openTab = async (url, tab) => {
+	const {newTabAlways} = await syncStore.getAll();
```

Comment:
why not fetch `newTabAlways` specifically?

Reply:
Because in the rest of the codebase it's `getAll` everywhere. I just want to stay consistent.

Reply:
okay, but it might be a good idea to go and fix such places someday, especially if such calls are inside event handlers or other UI-related code paths that can lead to lower perceived performance.